playground/frontend/playground_components/tools/extract_symbols_python/extract_python_symbols.py

```diff
@@ -0,0 +1,110 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+import argparse
+import ast
+import os
+import yaml
+from typing import Dict
+from typing import List
+
+
+def should_include_class_node(class_node: ast.ClassDef) -> bool:
```

Comment:
Is there a plan for unit tests for this method as well as the other methods?

Reply:
Done. I made a test for the entire command in Dart.

First I thought about testing Python code in Python, but it has a drawback. We will have 3 more scripts for this job in other languages, very much different inside but with an identical API. In Dart we can make 4 nearly identical tests instead of 4 very different native tests on those languages. So the Dart solution is easier to maintain.